### DISCUSSION TITLE
2 Commits behind most recent message

### DISCUSSION THREAD

**Discussion by krhyatt11** [qdm12/gluetun#1526] [Q&A]
In my Gluetun log -- it is showing this message (even if I have updated/redeployed the container) -- Running Portainer on a NAS:
INFO [vpn] You are running 2 commits behind the most recent latest

Any idea how to resolve this message?

**Comment by bnhf** [comment]
@krhyatt11 

Looks like the :latest tag isn't quite up-to-date -- I saw the same thing pulling the latest image. Using the :v3 tag works though:

```yml
    image: qmcgaw/gluetun:v3
```
